UNRELEASED_CHANGELOG.md

```diff
@@ -71,7 +71,12 @@
 ### ⬆️ Improved
 
 ### ✅ Added
-
+- Added self-contained higher-level UI components:
```

Comment:
Awesome work all around, very nicely done.

I have just one question: what about Docusaurus docs? 😀

Reply:
Was not able to complete the docs in the previous sprint so we have a task to do that in the current sprint